src/filesystem.rs

```diff
@@ -127,8 +127,8 @@ impl Filesystem {
     pub fn new(
         id: &str,
         author: &str,
-        resources_dir_name: &str,
-        resources_zip_name: &str,
+        resources_dir_name: &path::Path,
```

Comment:
Can you make these `impl Into<PathBuf>` too?

Reply:
We don't need to allocate a `PathBuf` here, so I used `AsRef<Path>` instead